ethics/README.md

```diff
@@ -0,0 +1,25 @@
+
+# Statement of ethics and rights
+
+Kanidm is a project that will store, process and present people's personal data. This means
+we have a responsibility to respect the data of all people who could be using our system -
+many who interact indirectly or do not have a choice in this platform.
+
+## Rights of people
+
+All people using this software should expect to have the right to:
+
+* Ability to be forgotten
+* Self determination and autonomy
+* Free from harmful discrimination of any kind
+* Informed consent and Privacy of their data
+* To use and access this software regardless of ability
```

Comment:
how about something like "to access and understand data held and shared on their behalf", intended to support access to their account audit trail with logins / information access / whatever else is created through their interaction with the service, in a supporting GDPR or NZ Privacy Act kinda way?